code/game/objects/items/devices/scanners.dm

```diff
@@ -540,7 +540,7 @@ Subject's pulse: ??? BPM"})
 		if(O.reagents.reagent_list.len)
 			for(var/datum/reagent/R in O.reagents.reagent_list)
 				var/reagent_percent = (R.volume/O.reagents.total_volume)*100
-				dat += "<br><span class='notice'>[R] [details ? "([R.volume] units, [reagent_percent]%)" : ""]</span>"
+				dat += "<br><span class='notice'>[R] [details ? "([R.volume] units, [reagent_percent]%[R.data && !istype(R,/datum/reagent/citalopram) && !istype(R,/datum/reagent/paroxetine) && !istype(R,/datum/reagent/incense) && !istype(R,/datum/reagent/antipathogenic) && !istype(R,/datum/reagent/vaccine) ? ", time in system: [R.data] seconds" : ""])" : ""]</span>"
```

Comment:
are you sure this is correct? I think this ticks only once every 2 seconds, and therefore time would be R.data*2

Reply:
You're right, which is why I just fixed this now.

Reply:
Strictly speaking it should be `SSmob.wait / 10` or something but nothing else in the game does that so whatever